include/rcutils/logging.h

```diff
@@ -116,45 +178,153 @@ rcutils_logging_output_handler_t rcutils_logging_get_output_handler();
 RCUTILS_PUBLIC
 void rcutils_logging_set_output_handler(rcutils_logging_output_handler_t function);
 
-/// The global severity threshold before calling the output handler.
+/// The default severity threshold for log calls.
 /**
- * The global severity threshold is being checked after the conditions when
- * using the various logging macros.
- *
- * \param severity The global severity threshold to be used.
+ * This severity threshold is used for nameless log calls and loggers with
+ * effective severity thresholds that are otherwise unspecified.
```

Comment:
Doesn't this also serve as the fallback when no specific level for a given name is specified?

Reply:
correct, that's what I meant by "loggers with effective severity thresholds that are otherwise unspecified". I'll re-word it.

Reply:
hopefully https://github.com/ros2/rcutils/pull/57/commits/6719f5da54f0e40e4c0fcd4ebf7a68044b2b5dec#diff-f1d39d4e841937fde043a8e49b7e6c2aR183 makes this clearer